1.8/administration/categories-and-forums.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: "Categories and Forums Management"
+categories: [administration]
+---
+
+## About Categories and Forums
+
+This is most basic guideline how to make and edit your first forums and categories at your forum.
+
+Categories and forums are root of your forum. It is suggested that they should be in some order so users and visitors can get good view of all categories and forums of your forum. There will be explained basic princip of how categories and forums work and what they are for.
+
+## Categories
+
+In Category you can't write anything, you can't open threads or post. They are made just for organisation. Inside them there are forums and sub-forums wich are here for threads and posting.
+
+
+### So what are Categories for then?
+
+Well, you can't just make some forum / subforum without it's parent. They cannot be alone. They needs parent Category in wich they are put. Categories are mostly general names of forums under it.
+For example, we have **category** named Community and under it **forums** like General Discussion, Showcase etc.
+
+### What Category can contain?
+
+Well, when creating new category they need to contain:
+- **Forum Type** - Forum type should be **category**.
+- **Title** - name of your category. Multiple categories can have same name. It is not it's id.
+- **Description** - You can have Category description. That can be few words wich will be displayed right below Category name at forum's home page. This field is **not** necessary and is mostly blank.
+- **Parent Forum** - Mostly this is left **None** as categories are here to actually be parents. But if you wish, you can put it under another category. Then it will look like forum but you will be unaviable to write in it so there is not much point in doing that.
+- **Display order** - This is position of your category between other categories. So if you have more than one category, there you can manage wich will be displayed at wich position. Please note that you can put any number in that places, and they will be sorted from lowest number - top category to biggest number - last category.
```

Comment:
I think this should be:

"**Display order** - This is the order of your category between other categories. So if you have more than one category, there you can manage which will be displayed at wich position. Please note that you can put any number in that places, and they will be sorted from lowest number (1) - top category to biggest number (100) - last category."